src/Widgets/AppContainers/AbstractAppContainer.vala

```diff
@@ -186,7 +186,7 @@ namespace AppCenter {
             open_button.clicked.connect (launch_package_app);
 
             button_grid = new Gtk.Grid ();
-            button_grid.column_spacing = 6;
+            button_grid.column_spacing = 0;
```

Comment:
```suggestion
```

You should be able to just remove this line instead of explicitly setting to `0`… right?